app/code/community/Bolt/Boltpay/Model/ShippingAndTax.php

```diff
@@ -180,10 +181,12 @@ public function getShippingAndTaxEstimate( Mage_Sales_Model_Quote $quote, $boltO
             $originalCouponCode = $quote->getCouponCode();
             if ($parentQuote) $quote->setCouponCode($parentQuote->getCouponCode());
             $this->boltHelper()->collectTotals(Mage::getModel('sales/quote')->load($quote->getId()), true);
```

Comment:
@mccottry is still require here?
As i see a bit below in `$this->applyShippingRate($quote, null, false);` we always recalculate quote Totals.

Reply:
Hi, @YevhenShBolt. When I make optimization changes, I try not to make speculative changes for no solid reason.  Those type of changes too often result in unwittingly introducing regression errors.  I researched the hi

Reply:
Hi, @YevhenShBolt. When I make optimization changes, I try not to make speculative changes for no solid reason.  For me, those type of changes too often result in unwittingly introducing regression errors.  

I researched the history of the line that you are focused on.  It originates back to some of the earliest PRs from @johnmccombs1 -  
#11 - Fixed bug where code wasn't populating weight and was causing discounts based on weight to not be applied.
https://github.com/BoltApp/bolt-magento1/pull/11/files
and
#20 Fixed issue where Taxjar wasn't getting loaded because we called $address->collectTotals before we called $quote->collectTotals. Collect totals gets cached and never loads Taxjar.
https://github.com/BoltApp/bolt-magento1/pull/20/files

@YevhenShBolt and @johnmccombs1 
I believe that this fix is still relevant.  Perhaps we just needed to add an internal comment to explain why to prevent someone in the future from removing it while under the belief that they are optimizing.  For this reason, I'll add a comment.

Reply:
@mccottry To add a comment, then we can merge this.